components/index.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const List = ({ items }) => {
```

Comment:
Hi Dennis, the list does not render in Vercel. I think you need to export the List "function".

Reply:
Thanks for your Review Volker :) My List is already inside of a List directory (as a index.js). 
Im not that sure, but i think that the Vercel-rendering is more important for the quality-assurance.